app/views/shared/_nav_left.html.erb

```diff
@@ -73,7 +73,7 @@
   <%= link_to(:HERBARIA.t, {controller: :herbarium, action: :index}, {class: "list-group-item indent"}) %>
   <%= link_to(:LOCATIONS.t, {controller: :location, action: :list_locations}, {class: "list-group-item indent"}) %>
   <%= link_to(:NAMES.t, {controller: :name, action: :observation_index}, {class: "list-group-item indent"}) %>
-  <%= link_to(:NEWS.t, {controller: :article, action: :list_articles}, {class: "list-group-item indent"}) %>
+  <%= link_to(:NEWS.t, articles_path, {class: "list-group-item indent"}) %>
   <%= link_to(:PROJECTS.t, {controller: :project, action: :list_projects}, {class: "list-group-item indent"}) %>
 
   <div class="list-group-item disabled bold">
```

Comment:
Can this be further DRYed/de-cluttered by iterating over a list of [title, href] pairs?

Reply:
Definitely. bootstrap-4 calls a partial for each item, but here you could just make an array and then iterate right in the view.

Reply:
From a coding standpoint, I'd greatly prefer using partials `_nav_left`. Just looking at the current code hurts my eyes. :-) 
But there's a performance question, especially with stuff that appears on nearly every page. I don't have the knowledge and time to investigate that now. And the question I raised is really outside the scope of this PR and story.
I created a new story. https://www.pivotaltracker.com/story/show/174142891
We should at least iterate over an array.

Reply:
I'd agree with @pellaea that if partials are more pleasing to code, we should not worry about the relative performance of them. I would agree, just going from analagous situations in other languages, that the performance hit is almost certainly pretty trivial. But we could benchmark it via a separate PR - why not?

(The biggest hits to MO performance that I found working on Bootstrap 4 are the expensive queries for name, naming and location that are run on observation#show. I believe having the production server running Ubuntu 16 is a considerable factor too).